databricks/koalas/plot.py

```diff
@@ -51,7 +52,7 @@ def _get_standard_kind(kind):
 
 
 class TopNPlot:
-    max_rows = 1000
+    max_rows = get_option("plotting.max_rows")
```

Comment:
Oh! this shouldn't be here. It will statically assign the value into the class value `TopNPlot. max_rows `. `get_option("plotting.max_rows")` should be called within each method, `get_top_n` and `set_result_text`.

Reply:
yes!! i also found this caused failure everytime... i am refactoring ^^

thanks!